src/Endatix.Core/UseCases/Submissions/SubmissionDto.cs

```diff
@@ -0,0 +1,54 @@
+using System.Reflection.Metadata.Ecma335;
+using System.Text.Json;
+using Ardalis.GuardClauses;
+using Endatix.Core.Entities;
+
+namespace Endatix.Core.UseCases.Submissions;
+
+/// <summary>
+/// Represents a submission data transfer object.
+/// </summary>
+/// <param name="Id">The unique identifier of the submission.</param>
+/// <param name="IsComplete">Indicates if the submission is complete.</param>
+/// <param name="JsonData">A dictionary containing JSON data related to the submission.</param>
+/// <param name="FormDefinitionId">The unique identifier of the form definition associated with the submission.</param>
+/// <param name="CurrentPage">The current page of the submission, if applicable.</param>
+/// <param name="CompletedAt">The date and time when the submission was completed, if applicable.</param>
+/// <param name="CreatedAt">The date and time when the submission was created.</param>
+public record SubmissionDto(long Id, bool IsComplete, Dictionary<string, object> JsonData, string Metadata, long FormDefinitionId, int? CurrentPage, DateTime? CompletedAt, DateTime CreatedAt)
+{
+    public long Id { get; init; } = Id;
+    public bool IsComplete { get; init; } = IsComplete;
+    public Dictionary<string, object> JsonData { get; init; } = JsonData;
```

Comment:
Yes, the idea is to send it parsed to the webhook consumer. Otherwise it will be another parsing they need to do just for the JSON data. This is the result with this change (otherwise it will be a string of the stringified JSON data they will need to deserialize)

![image](https://github.com/user-attachments/assets/e7d048f7-b37a-4cbd-801e-eb55f6b3854a)